app/src/providers/loggerProvider.tsx

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: BUSL-1.1; Copyright (c) 2025 Social Connect Labs, Inc.; Licensed under BUSL-1.1 (see LICENSE); Apache-2.0 from 2029-06-11
+
+import React, { createContext, useContext } from 'react';
+
+import * as Logger from '../utils/logger';
```

Comment:
nit: let's try and avoid * imports and exports and direct the methods we need directly. this helps with tree shaking

see that you're wrapping it with `createContext` so it's probably fine here